language/move-vm/runtime/src/loader.rs

```diff
@@ -1636,9 +1654,6 @@ impl Module {
 // (rather then "compiled") to make available data needed for execution
 // #[derive(Debug)]
 struct Script {
-    // primitive pools
-    script: CompiledScript,
```

Comment:
Was this just unused?

Reply:
It was only used to fetching constants. Now that I have a replacement, it's no longer needed.